LocalSettings.php

```diff
@@ -2381,23 +2379,16 @@
 			'managewiki-editdefault',
 			'mwoauthmanageconsumer',
 			'mwoauthmanagemygrants',
-			'mwoauthproposeconsumer',
 			'mwoauthsuppress',
-			'mwoauthupdateownconsumer',
 			'mwoauthviewprivate',
 			'mwoauthviewsuppressed',
-			'read',
 			'renameuser',
 			'requestwiki',
 			'siteadmin',
-			'skipcaptcha',
```

Comment:
allows people to disable captcha on the whole wiki.

Reply:
@JohnFLewis some wikis might only want CAPTCHA for anonymous users and they may want to disable it on the whole wiki for any user who has an account, whether it’s an attached global account or an independent locally created account